Lesson13/Kubra/conditionalStatements.js

```diff
@@ -89,57 +151,158 @@ election2(14);
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
 
+function exercise8(height, weight){
+  if (typeof weight === 'number' && typeof height === 'number' && height > 0 && weight > 0) {
+    const bmi = weight / (height * height);
+
+    if (bmi < 18.5) {
+      console.log("Underweight")
+    } else if (bmi >= 18.5 && bmi <= 24.9) {
+      console.log("Healthy Weight")
+    } else if (bmi >= 25.0 && bmi <= 29.9) {
+      console.log("Overweight")
+    } else {
+      console.log("Obesity")
+    }
+  } else {
+    return "Please provide valid weight (in kg) and height (in meters).";
+  }
+}
+
 // 9. Write a program that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
 
+function exercise9(age){
+  if(typeof age === 'number' && age>=0){
+    if (age <= 12) {
+      console.log('Child');
+    } else if (age <= 19) {
+      console.log('Teenager');
+    } else if (age <= 64) {
+      console.log('Adult');
+    } else {
+      console.log('Senior Citizen');
+    }
+  } else{
+    console.log("Please provide a valid number for age.")
+  }
+}
+
 // 10. Write a program that asks the user to enter their age and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
 
+function exercise10(age, isStudent) {
+  if (age < 18 && isStudent === 'yes') {
```

Comment:
In programming, if you have a value that is something like 'yes'/'no', it's common to use a boolean type value for it. So the code would change like this:
```suggestion
  if (age < 18 && isStudent) {
```